tests/scripts/local_monitoring_tests/rest_apis/test_namespace_reco_e2e_workflow.py

```diff
@@ -254,11 +246,8 @@ def test_list_recommendations_namespace_exps(cluster_type):
     assert data['status'] == SUCCESS_STATUS
     assert data['message'] == CREATE_EXP_SUCCESS_MSG
 
-    # Wait for the container to complete
-    wait_for_container_to_complete(container_id1)
-    wait_for_container_to_complete(container_id2)
-    wait_for_container_to_complete(container_id3)
-
+    # Wait for the threshold for short term recommendations
+    time.sleep(180)
```

Comment:
Please ensure the test behavior is consistent and short recommendations are generated after this

Reply:
@shekhar316 - Any updates on this?